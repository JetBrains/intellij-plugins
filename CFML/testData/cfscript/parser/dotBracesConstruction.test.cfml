<cfscript>
request.get("test");
</cfscript>