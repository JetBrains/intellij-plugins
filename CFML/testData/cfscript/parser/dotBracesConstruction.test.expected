CfmlFile:dotBracesConstruction.cfml
  Tag
    PsiElement(OPENER)('<')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(R_ANGLEBRACKET)('>')
    FunctionCallExpression
      ReferenceExpression
        PsiElement(SCOPE_KEYWORD)('request')
        PsiElement(POINT)('.')
        PsiElement(IDENTIFIER)('get')
      ArgumentList
        PsiElement(L_BRACKET)('(')
        StringLiteral
          PsiElement(DOUBLE_QUOTE)('"')
          PsiElement(STRING_TEXT)('test')
          PsiElement(DOUBLE_QUOTE_CLOSER)('"')
        PsiElement(R_BRACKET)(')')
    PsiElement(SEMICOLON)(';')
    PsiElement(LSLASH_ANGLEBRACKET)('</')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(CLOSER)('>')